docs/list-and-array.md

```diff
@@ -96,3 +96,5 @@ The above array access/update is just syntax sugar for `Array.get`/`Array.set`.
 ### Tips & Tricks
 
 If you're compiling to JavaScript, know that Reason arrays map straightforwardly to JavaScript arrays, and vice-versa. Thus, even though arrays are fix-sized on native, you can still use the `Js.Array` API to resize them. This is fine.
+
+If you're coming from JavaScript, iterables behave differently. To get the more familiar callback signature of `(element, index) => {}`, you would use `.mapi()` or `.iteri()`. If you just need an element in the callback, use `.map()`. See the [API reference](https://reasonml.github.io/api/Array.html#VALmapi).
```

Comment:
Javascript's `.map()`'s callback has an argument signature of `(element[, index,[array]])`. Reason's `.mapi()` callback has an argument signature of `(index, element)`. It's more familiar to the JS, but the order is reversed, and there is no array.

Reply:
@malisbad Good point